README.adoc

```diff
@@ -45,31 +44,32 @@ Other configuration options include:
 * Switches (options) to provide to the Gradle execution.
 * Tasks to execute (if blank the defaults tasks of the build will be invoked).
 * Path to the build script if different from the root directory of the build.
-* Name of the build script if different from build.gradle.
+* Name of the build script if different from `build.gradle`.
```

Comment:
Above (can't link that specific line because it's not modified) there is
`The system provides both automatic installation, which can be performed by directly downloading from the http://www.gradle.org/[Gradle web site], extracting a compressed final or executing some shell commands.`
What does `compressed final` means ?

Reply:
Adapted